ui/src/app/views/settings/status/status.html

```diff
@@ -2,20 +2,26 @@
     <h3>{{'admin_status_title' | translate }}</h3>
 
     <div *ngIf="loading" class="ui active text loader">{{ 'common_loading' | translate }}</div>
+    
+    <div class="ui five wide column" *ngIf="status && !loading">
+        <div class="ui fluid input">
+            <input type="text" placeholder="{{ 'filter' | translate }}" [(ngModel)]="filter">
+        </div>
+    </div>
 
     <table class="ui definition table" *ngIf="status && !loading">
         <tbody>
-        <tr *ngFor="let line of status.lines" [ngClass]="{'error': line.status=='AL', 'warning': line.status=='WARN', 'positive': line.status=='WARN'}">
-            <td>
+            <tr *ngFor="let line of getStatusLines()" [ngClass]="{'error': line.status=='AL', 'warning': line.status=='WARN', 'positive': line.status=='WARN'}">
```

Comment:
strict equals `===`

Reply:
avoid using function in ngIf/ngFor etc..